source/content/guides/secrets/02-basic-concepts.md

```diff
@@ -0,0 +1,86 @@
+contributors: [stovak]
+contenttype: [guide]
+innav: [true]
+categories: [secrets]
+cms: [drupal, wordpress]
+audience: [development]
+product: [secrets]
+integration: [--]
+tags: [reference, cli, local, terminus, workflow]
+permalink: docs/guides/secrets/basic-concepts
+reviewed: "2024-05-01"
+---
+<dl>
+
+<dt>Secret</dt>
+
+<dd>
+
+A key-value pair that should not be exposed to the general public, typically something like a password, API key, or other sensitive information that should not be added to version control.
+
+</dd>
+
+<dt>Secret type<sup>1</sup></dt>
+
+<dd>
+This represents how the secret is used.  A secret can only have one type.
+
+Current types are:
+
+  * `runtime`: This secret type can be retreived directly from your application code.  This is the recommended type if you want your application to be able to use the secret while it's operating.
+
+  * `env`: This type is used to set environment variables. Environment variables are currently only supported for Integrated Composer builds; setting environment variables on the application server is unsupported.
+
+  * `composer`: This secret type is specifically used for authentication when pulling Composer packages from private repositories.  This is the recommended method for installing private composer packages.
+
+</dd>
+
+<dt>Secret Scope</dt>
+
+<dd>
+A secret's scope is the answer to the question "Where is the secret's value available?". Once set, a secret's scope cannot be changed. The secret must be deleted and recreated.
+
+  * `ic`: This secret will be readable by the Integrated Composer runtime. You should use this scope to get access to your private repositories.
+
+  * `web`: this secret will be readable by the application runtime.
+
+  * `user`: this secret will be readable by the user. This scope should be set if you need to retrieve the secret value at a later stage.
+
+</dd>
+
+<dt>Owning Entity</dt>
+
+<dd>
+
+* **Organization-owned secrets** - Organization-owned secrets are available to every site and environment that are associated with the owning organization. A common use-cases is for a CI system and infrastructure that's shared among all sites in an organization. Note that secrets from "Supporting" Organizations are explicitly ***not shared*** with the sites they support. Sites receive secret key/value pairs from their Primary Organization only.
+
+* **Site-owned secrets** - Site-owned secrets are available to the site and all of its environments. A common use-case is Github tokens that a site's composer build can use to access private repos referenced in the composer file.
+
+* **Environment override** - Environment overrides provide overrides to a secret value for a specific environment. A common use case for this are API keys that are different in production and non-production environments.
+
+![Secrets Relationships](../../../images/guides/secrets/secrets-relationships.png)
```

Comment:
Maybe we could replace the diagram with something simpler, similar to this diagram:
![Screenshot 2024-07-23 at 11 22 10 AM](https://github.com/user-attachments/assets/cb81504a-37e5-4a32-bb1d-7abb36579138)
Perhaps this goes in the "Value Resolution" section.

Reply:
Want to use that exact screenshot?  I'm thinking maybe it's a bit wide and maybe we could put the 2nd table below the first?

Reply:
I was hoping someone would draw a more artistic version. 😄  If the tables are vertically positioned, then maybe the vertical and horizontal axis should also be reversed.

Reply:
I think we could also simply remove the diagram, and rely solely on the text, if we don't want to redraw it per the proposal above.

Reply:
@mel-miller is this something you can help with?

Reply:
@rachelwhitton — We probably shouldn't have this image since it is not accessible (unless the alt text can reference a text-based description of this or the article contains this same thing in text and we can mark this as decorative). 

If you want to keep an image but have a new one drawn you can reach out to creative services (RJ Bravo) to request it.